src/components/navbar.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react';
+import './navbar.css';
+
+import Dashboard from './dashboard';
+import OpenIssues from './open';
+import ResolvedIssues from './resolved';
+import NewIssue from './new';
+
+import { BrowserRouter as Route, NavLink } from 'react-router-dom';
+
+
+export default function NavBar() {
+    const [nav] = useState();
+    return (
+      <main>
+        <div className='linkInfo'>
+          <NavLink className='dashboard navBarLink' to='/'>Dashboard</NavLink>
+          <NavLink className='open navBarLink' to='/open'>Open Issues</NavLink>
+          <NavLink className='resolved navBarLink' to='/resolved'>Resolved Issues</NavLink>
+          <NavLink className='new navBarLink' to='/new'>New Issue</NavLink>
+        </div>
```

Comment:
The Navlink section looks fine, I might change the wrapping divs to `ul`.  What I would definitely change is to remove the Routes from the Navbar and put them inside App.js